src/sentry/integrations/repository/metric_alert.py

```diff
@@ -34,45 +40,33 @@ def from_model(cls, instance: NotificationMessage) -> MetricAlertNotificationMes
         )
 
 
-class NewMetricAlertNotificationMessageValidationError(Exception):
+class NewMetricAlertNotificationMessageValidationError(NotificationMessageValidationError):
```

Comment:
are we subclassing this to make the relationship explicit? because the only thing we do with this is subclass it again to create `IncidentAndTriggerActionValidationError`?

Reply:
We will use this for the issue alert validation errors too, so for now it looks single use but it will be used again too. Hoping it can be _the_ error we raise for any validation error in this domain area, so if that base error is ever seen, we know it's for this code/domain area